packages/localize/src/localize.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2017 Ericson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+
+import { MessageTranslation } from './message-translation';
```

Comment:
this is the same as the generated file ? why is it there?

Reply:
The file is needed to run the test and I cannot generate the file at the same time as compiling the code !!!. Also, the file is different for the import section